.test-infra/jenkins/job_PerformanceTests_Debezium.groovy

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as common
+import Kubernetes
+import InfluxDBCredentialsHelper
+
+String jobName = "beam_PerformanceTests_Debezium"
+
+// String kubernetesYmlPath = "src/.test-infra/kubernetes/postgres/\n" +
+//                            "postgres-service-for-debezium.yml"
+
+
+String task = ":sdks:python:apache_beam:testing:load_tests:run -PloadTest.mainClass=apache_beam.testing.load_tests.debezium_performance -Prunner=DirectRunner"
+
+static String parseOptions(Map<String, ?> options) {
+options.collect { entry ->
+  // Flags are indicated by null values
+  if (entry.value == null) {
+    "--${entry.key}"
+  } else {
+    "--${entry.key}=$entry.value".replace('\"', '\\\"').replace('\'', '\\\'')
+  }
+}.join(' ')
+}
+
+job(jobName) {
+  common.setTopLevelMainJobProperties(delegate)
+  common.setAutoJob(delegate, 'H H/12 * * *')
+  common.enablePhraseTriggeringFromPullRequest(
+        delegate,
+        'Python Debezium Performance Test',
+        'Run Python Debezium Performance Test')
+  InfluxDBCredentialsHelper.useCredentials(delegate)
+
+//   String namespace = common.getKubernetesNamespace(jobName)
+//   String kubeconfig = common.getKubeconfigLocationForNamespace(namespace)
+  String postgresHostName = "LOAD_BALANCER_IP"
+
+//   Kubernetes k8s = Kubernetes.create(delegate, kubeconfig, namespace)
+//   k8s.apply(common.makePathAbsolute(kubernetesYmlPath))
+//   k8s.loadBalancerIP("postgres-for-dev", postgresHostName)
+
+  //(0..2).each { k8s.loadBalancerIP("outside-$it", "Postgres_IP_$it") }
+
+  Map pipelineOptions = [
+    //kubernetes_host : "\$LOAD_BALANCER_IP",
+    kubernetes_host : '146.148.98.7',
+    kubernetes_port : "5432",
+    postgres_user: 'postgres',
+    postgres_password: 'uuinkks',
+    input_options  : '\'{' +
+        '"num_records": 20000000 }\''
+  ]
+
+
+  steps {
+    gradle {
+      rootBuildScriptDir(common.checkoutDir)
+      dependsOn("sdks:java:io:debezium:expansion-service:shadowJar")
```

Comment:
The seed job failed, it doesn't like this line - https://ci-beam.apache.org/job/beam_SeedJob/10045/console